luigi/contrib/bigquery.py

```diff
@@ -33,6 +33,8 @@
     logger.warning('Bigquery module imported, but google-api-python-client is '
                    'not installed. Any bigquery task will fail')
 
+IMPORT_TIME_SALT = random.randrange(0, 1e10)
```

Comment:
I think I gave that idea - I thought it might be better given the way luigi targets tend to get re-constructed (which would cause two BigqueryTargets pointing to the same table to have two different temp table names). It feels as though maybe the nicest way would be to either have a completely consistent table name (but one that may exist so maybe blow it away at use time), or return a new table name every time the function is called.

But really, I think any of these approaches are fine :)